test/Ocelot.UnitTests/Configuration/Validation/FileConfigurationFluentValidatorTests.cs

```diff
@@ -1564,11 +1564,17 @@ private class TestOptions : AuthenticationSchemeOptions
         }
 
         private class TestHandler : AuthenticationHandler<TestOptions>
-        {
+        {
+#if NET8_0_OR_GREATER
+            public TestHandler(IOptionsMonitor<TestOptions> options, ILoggerFactory logger, UrlEncoder encoder) : base(options, logger, encoder)

```

Comment:
Why was `ISystemClock clock` param removed?

Reply:
ISystemClock is deprecated in .NET8

Reply:
But what is an alternative in .NET 8?

Reply:
TimeProvider is now a settable property on the Options classes for the authentication and identity components. It can be set directly or by registering a provider in the dependency injection container.

Reply:
And I don't think we were using ISystemClock anyway...

Reply:
But how did you apply [Recommended action](https://learn.microsoft.com/en-us/dotnet/core/compatibility/aspnet-core/8.0/isystemclock-obsolete#recommended-action) ?

Reply:
just by using the new constructor + public BasicAuthenticationHandler(IOptionsMonitor<AuthenticationSchemeOptions> options, ILoggerFactory logger, UrlEncoder encoder)
+     : base(options, logger, encoder)